lang/proto/proto.talon

```diff
@@ -0,0 +1,19 @@
+mode: command
+and mode: user.protobuf
+mode: command
+and mode: user.auto_lang
+and code.language: protobuf
+-
+
+state message: "message "
+state package: "package "
+state reserved: "reserved "
+state enum: "enum "
+op equals: " = "
+state import: "import "
+state import public: "import public "
+state option: "option "
+state repeated: "repeated "
+
+<user.code_insert_type>$:
```

Comment:
This should probably lose the end-anchor. It's also probably better to use the list of types directly rather than a weird capture, so you could do:

```
type {user.code_type}: "{code_type}"
repeated type {user.code_type}: "repeated {code_type}"
```